index.html

```diff
@@ -36,13 +38,14 @@ <h2>Location and Time</h2>
   </section>
   <section id="dates">
     <h2>Important Dates</h2>
-    <p>Due to low level of submissions, the Program Committee is re-evaluating the timing and format of the workshop. This page will be updated once these new plans emerge - please let <a href="mailto:group-secure-the-web-forward@w3.org">group-secure-the-web-forward@w3.org</a> know if you want to be alerted.</p>
+    <p>Due to low level of submissions, the Program Committee has re-evaluated the timing and format of the workshop. The workshop will now be primarily virtual across 3 two-hour virtual sessions held on the 26th, 27th and 28th of September. In addition we will be holding open in-person sessions at W3C's TPAC event in Valencia, Spain on the 13th of September (time tbd) and at the Open Source Summit Europe event in Bilbao, Spain, the week of September 18th (exact date and time tbd). New dates are as follows:</p>
     <del>
     <ul>
-      <li><Position papers due: May 2nd (extended from <del>Apr 24</del>)</li>
-      <li>Invitations sent to participants: May 8</li>
-      <li>Program Announced: May 22</li>
-      <li>Workshop: June 7-8</li>
+      <li><Position papers due: July 28th</li>
+      <li>Invitations sent to participants: August 4th</li>
+      <li>Program Announced: August 25th</li>
+      <li>In-person sessions: September 13th and week-of September 18th (date tbd)</li>
```

Comment:
this should be clarified that these will be informal sessions to give a taste of submissions